vertx-pg-client/src/main/java/io/vertx/pgclient/impl/codec/DataType.java

```diff
@@ -186,8 +186,11 @@ static DataType lookup(Class<?> type) {
     for (DataType dataType : values()) {
       oidToDataType.put(dataType.id, dataType);
     }
-    encodingTypeToDataType.put(String.class, VARCHAR);
-    encodingTypeToDataType.put(String[].class, VARCHAR_ARRAY);
+//    encodingTypeToDataType.put(String.class, VARCHAR);
```

Comment:
why is this commented ?

Reply:
As far as I understand it, there can only be one mapping per class to a Type. This is the Problem i was talking about in the other comments which we need to solve here - so either there will be a need for a wrapper class or someone can come up with a better solution for this

Reply:
you are totally right, this is because we cannot specify which database type to use in a tuple for a given java type

Reply:
So there will be something like a new Class like `PgXMLType` which just holds a String. Would you agree to that?